.travis.yml

```diff
@@ -92,6 +92,14 @@ jobs:
         # warnings to the .pylintrc-wip file to prevent them from being
         # re-introduced
         - pylint --rcfile=.pylintrc-wip can/
+    - stage: linter
+      name: "Formatting Checks"
+      python: "3.7"
+      before_install:
+        - travis_retry pip install -r requirements-lint.txt
+      script:
+        - black --diff .
+        - black --check .
```

Comment:
I could image `black --check --verbose .` being enough, [since black itself uses it](https://github.com/python/black/blob/master/.travis.yml#L25)